tyler_thomson/1955API/server/controllers/persons.js

```diff
@@ -0,0 +1,44 @@
+let mongoose = require('mongoose');
+let Person = mongoose.model('Person');
+
+module.exports = {
+  index: function(req, res){
+    Person.find({}, function(err, person){
+      return res.json(person);
+    })
+  },
+  create: function(req, res){
+    Person.create({name: req.params.name}, function(err, person){
+      if(err){
+        console.log(err);
+        }
+      else{
+        console.log("Person added to database successfully")
+        console.log(person);
+        return res.redirect('/');
+      }
+    })
+  },
```

Comment:
When making an API we want to return exclusively JSON.  So, even on post requests it would be best to respond with JSON.  The newly created item is a good idea to return from create functions.  Also, if you get errors you want to return the error object as JSON, too.   Using the following pattern for your back-end controllers will be helpful...
``` javascript
create: function(req, res){
    Person.create({ name: req.params.name }, function(err, person){
        if(err) { return res.json(err) }
        else { return res.json(person) }
    }
}
```